microsoft-edge/devtools-guide-chromium/customize/index.md

```diff
@@ -66,19 +61,32 @@ You can move tools between the main panel and the drawer.
     :::image type="complex" source="../media/move-to-drawer.msft.png" alt-text="Move tool from the main panel to the Drawer" lightbox="../media/move-to-drawer.msft.png":::
        Move tool from the main panel to the **Drawer**
     :::image-end:::  
-    
 
 ## Reorder panels  
 
 Select and drag a tool to change the ordering.  Your custom tool order persists across DevTools sessions.  
 
 > [!NOTE]
-> By default, the **Network** tool is usually the fourth from the left.  In the following figure, the **Network** tool is the first from the left.  
+> By default, the **Network** tool is usually the fifth from the left.  In the following figure, the **Network** tool is being moved to the first position from the left.  
 
 :::image type="complex" source="../media/customize-network-first-position.msft.png" alt-text="Custom order of Devtools in a panel" lightbox="../media/customize-network-first-position.msft.png":::
    Custom order of Devtools in a panel  
 :::image-end:::  
 
+## Open and close panels  
```

Comment:
Consider retitling to "Open and close tools".  Give hints or explicit discussion, that tab ~= tool ~= panel (~= pane).  A tab (as in, **More tabs** (>>)) is a common, concrete UI item meaning both the little rect & pane.  In the Command Menu, tools are called _panels_.  Practically, users need to know both terms.